storage_devices/disk.py

```diff
@@ -269,15 +276,21 @@ def get_pci_address(cls, device_id):
 
         return pci_address[-1]
 
+    @classmethod
+    def identify(cls, device_name: str) -> bool:
+        output = TestRun.executor.run(
+            f"realpath /sys/block/{device_name}/device/driver | grep nvme"
+        )
+        return output.exit_code == 0
+
 
 class VirtioDisk(Disk):
     plug_all_command = "echo 1 > /sys/bus/pci/rescan"
 
     def __init__(self, path, disk_type, serial_number, block_size):
         Disk.__init__(self, path, disk_type, serial_number, block_size)
         self.plug_command = VirtioDisk.plug_all_command
-        self.unplug_command = \
-            f"echo 1 > {self.get_unplug_path(self.get_device_id())}"
+        self.unplug_command = f"echo 1 > {self.get_unplug_path(self.get_device_id())}"
         self.pci_address = VirtioDisk.get_pci_address(self.get_device_id())
 
     @classmethod
```

Comment:
As you implemented new type recognition mechanism, `get_unplug_path()` , `get_pci_address()` and `get_sysfs_addr()` don't have be classmethods anymore

Reply:
I mean it for all disk types

Reply:
I mean for disk types all disk types these 3 methods must be thoroughly reviewed and refactored.
1. Make sure if the are needed 
2. Make sure that all relevant checks are called from constructors instead of these functions